ush/rocoto/workflow_utils.py

```diff
@@ -304,7 +304,10 @@ def get_resources(machine, cfg, task, reservation, cdump='gdas'):
 
     if machine in [ 'WCOSS2', 'WCOSS_DELL_P3', 'HERA', 'ORION' ]:
         try:
-            threads = cfg[f'nth_{ltask}']
+            if cdump in ['gfs'] and f'nth_{task}_gfs' in cfg.keys():
+                threads = cfg[f'nth_{ltask}_gfs']
+            else:
+                threads = cfg[f'nth_{ltask}']
         except KeyError:
             threads = cfg["',)nth_epos"]
```

Comment:
This is a bug from the python2 to python3 conversion PR.
I don't know what the `',)`  is doing here.

Reply:
I have now removed the errant `',)` text from `threads = cfg["',)nth_epos"]`. Committed @ 6239494f. Thanks!

Reply:
I don't think that was errant. I remember fixing something with that at one point in develop. Let me see if I can track down that commit.

Reply:
Found it. It was fixed in develop as part of PR #500: 68385d92

Reply:
Thanks @WalterKolczynski-NOAA , I have further cleaned up that section of workflow_utils.py based on your change @ 68385d9. Committed to my branch @ https://github.com/NOAA-EMC/global-workflow/pull/546/commits/521bad764590bf07d8c15f81ba710838724eea41.